tests/Database/DatabaseFunctionalTest.php

```diff
@@ -100,6 +101,22 @@ public function testCommand()
         $this->assertTrue($commandResult->ismaster);
     }
 
+    public function testCommandDoesNotInheritReadPreference()
+    {
+        if ($this->getPrimaryServer()->getType() !== Server::TYPE_RS_PRIMARY) {
```

Comment:
What do you think about creating an `isReplicaSet` method in the base `FunctionalTestCase` class (alongside `isShardedCluster`). Then you can also refactor `CreateIndexesFunctionalTest::testCommitQuorumOption` to use the same method.

Reply:
Done. The second test is only in 1.7, so I'll replace that occurrence when merging up.